src/lib/state/DY.Lib.State.PKI.fst

```diff
@@ -38,19 +40,18 @@ type pki_key = {
 %splice [ps_pki_key] (gen_parser (`pki_key))
 %splice [ps_pki_key_is_well_formed] (gen_is_well_formed_lemma (`pki_key))
 
-type pki_value (bytes:Type0) {|bytes_like bytes|} = {
+[@@ with_bytes bytes]
```

Comment:
What happened to the `bytes_like bytes` requirement? Is this annotation saying the same thing?

Reply:
Before this change, `pki_value` would be instantiated with any `bytes_like` instance (so, for example, lists of u8). Because `bytes` is a parameter to it, if we want to talk about the `pki_value` intantiated with the symbolic `bytes` of DY*, we needed to name it `pki_value bytes`.

With the change of the interface for `map`, I needed to write `pki_value` in a few places, in its full form (`pki_value bytes`). I found it annoying, given the fact that it will only be instantiated with DY*'s `bytes`. So I removed the `bytes` parameter, so that we can simply write `pki_value`, and replaced the `bytes` + `bytes_like` parameters with an annotation saying to Comparse "generate a parser / serializer only using DY*'s `bytes`".

It's not a regression in expressivity because `pki_key` have the same restriction anyway, I don't know why `pki_value` in particular was using a generic `bytes`.

Reply:
Can't we leave it generic (in both places, for `pki_value` and `pki_key`) and define some shorthands for the DY* `bytes` versions? (Assuming the reason for this change really was just laziness.)

Reply:
I see. Ok then let's just use teh DY* `bytes` instances everywhere.